components/modal/ModalEthWrapper.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="inline-block">
+    <slot>
+      <a class="underline cursor-pointer" @click="show = true"
+        >Click here to wrap ETH</a
+      >
```

Comment:
```suggestion
      <a class="underline cursor-pointer" @click="show = true">
           Click here to wrap ETH
     </a>
```

Could you fix the indentation to look like this?, also happen in other tags in this file. its easier to read. 
For some reason lint is allowing your way.